packages/storybook/cxl-ui/cxl-details/cxl-details-checkout.stories.js

```diff
@@ -0,0 +1,31 @@
+import '@conversionxl/cxl-lumo-styles';
+import '@conversionxl/cxl-ui/src/components/cxl-details/cxl-details-checkout.js';
+import { html } from 'lit';
+
+export default {
+  args: {
+    theme: '',
+  },
+  argTypes: {
+    theme: {
+      options: ['filled', 'small', 'reverse'],
+      control: { type: 'select' },
+    },
+  },
+  parameters: {
+    layout: 'centered',
+  },
+  title: 'CXL UI/cxl-details',
+};
+
+// eslint-disable-next-line no-empty-pattern
+const Template = ({ theme }) => html`
+  <cxl-details-checkout opened summary-field="sampleInput" theme=${theme}>
+    <div slot="summary">Summary<span class="summary-field" hidden> - <span class="summary-field-value"></span></span></div>
```

Comment:
* try `<vaadin-horizontal-layout>` here instead of plain spans
* try separating step summaries with something like `&mdash;` &mdash;